WindowsServerDocs/networking/technologies/ipam/ipam-top.md

```diff
@@ -15,14 +15,14 @@ author: shortpatti
 ---
 # IP Address Management (IPAM)
 
->Applies to: Windows Server (Semi-Annual Channel), Windows Server 2016
+> Applies to: Windows Server (Semi-Annual Channel), Windows Server 2016
 
-IP Address Management (IPAM) is an integrated suite of tools to enable end-to-end planning, deploying, managing and monitoring of your IP address infrastructure, with a rich user experience. IPAM automatically discovers IP address infrastructure servers and Domain Name System (DNS) servers on your network and enables you to manage them from a central interface.  
-  
-> [!NOTE]  
-> In addition to this topic, the following IPAM content is available.  
->   
-> -   [What's New in IPAM](../../technologies/ipam/What-s-New-in-IPAM.md)  
-> -   [Manage IPAM](../../technologies/ipam/Manage-IPAM.md)  
-> -   [IP Address Management (IPAM) Server Cmdlets in Windows PowerShell](https://technet.microsoft.com/library/jj553807.aspx)  
-> -   Video: [Windows Server 2016: DNS management in IPAM](https://channel9.msdn.com/Blogs/windowsserver/Windows-Server-2016-DNS-management-in-IPAM)  
\ No newline at end of file
+IP Address Management (IPAM) is an integrated suite of tools to enable end-to-end planning, deploying, managing and monitoring of your IP address infrastructure, with a rich user experience. IPAM automatically discovers IP address infrastructure servers and Domain Name System (DNS) servers on your network and enables you to manage them from a central interface.
+
+> [!NOTE]
+> In addition to this topic, the following IPAM content is available.
+>
+> - [What's New in IPAM](../../technologies/ipam/What-s-New-in-IPAM.md)
+> - [Manage IPAM](../../technologies/ipam/Manage-IPAM.md)
+> - [IP Address Management (IPAM) Server Cmdlets in Windows PowerShell](https://docs.microsoft.com/en-us/powershell/module/ipamserver/?view=win10-ps)
```

Comment:
```suggestion
> - [IP Address Management (IPAM) Server Cmdlets in Windows PowerShell](https://docs.microsoft.com/powershell/module/ipamserver/?view=win10-ps)
```
It's better off removing the locale so when the localized version of this documentation is available, it will redirect you that version instead of the English one.